lib/internal/util.js

```diff
@@ -514,12 +514,15 @@ function isInsideNodeModules() {
   return false;
 }
 
-function once(callback) {
+function once(callback, preserveReturnValue = false) {
```

Comment:
Would prefer options instead of a boolean so it's clear what it is on the call site